packages/adminrouter/extra/apache-windows/adminrouter.conf

```diff
@@ -0,0 +1,143 @@
+LoadModule setenvif_module modules/mod_setenvif.so
+
+<IfModule unixd_module>
+User daemon
+Group daemon
+</IfModule>
+
+DocumentRoot "c:\opt\mesosphere\bin\Apache24\htdocs"
+
+<Directory />
+    AllowOverride none
+    Require all denied
+</Directory>
+
+# Configures the footer on server-generated documents
+ServerSignature Off
+
+# Configures to send just "Server:Apache" in the Server HTTP response header 
+ServerTokens Prod
+
+<VirtualHost *:61001>
+    # Enforce 1024M request body max size limit for DC/OS 
+    LimitRequestBody 1073741824
+    # Set keepalive connections timeout for DC/OS
+    KeepAliveTimeout 65
+    # Settings for Apache proxy options
+    ProxyPreserveHost On
+    # Disable forward proxy
+    ProxyRequests Off
+    <Proxy *>
+        Order allow,deny
+        Allow from all
+    </Proxy>
+
+    # Proxy routing for DC/OS Metrics service
+    <Location /system/v1/metrics>
+        ProxyPass        http://localhost:9000
+        ProxyPassReverse http://localhost:9000
+    </Location>
+
+    # Proxy routing for DC/OS Diagnostics service
+    <Location /system/health/v1>
+        ProxyPass        http://localhost:9003/system/health/v1
+        ProxyPassReverse http://localhost:9003/system/health/v1
+    </Location>
+    # Proxy routing for DC/OS Pkgpanda service
+    <Location /pkgpanda>
+        ProxyPass        http://localhost:9001
+        ProxyPassReverse http://localhost:9001 
+    </Location>
+    # Routing for DC/OS Logging service
+    <Location /system/v1/logs>
+        ProxyPass        http://localhost:9002
+        ProxyPassReverse http://localhost:9002
+    </Location>
+
+</VirtualHost>
+
+<IfModule dir_module>
+    DirectoryIndex index.html
+</IfModule>
+
+<Files ".ht*">
+    Require all denied
+</Files>
+
+ErrorLog "logs/error.log"
+LogLevel warn
+
+<IfModule log_config_module>
+    LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\"" combined
+    LogFormat "%h %l %u %t \"%r\" %>s %b" common
+
+    <IfModule logio_module>
+      LogFormat "%h %l %u %t \"%r\" %>s %b \"%{Referer}i\" \"%{User-Agent}i\" %I %O" combinedio
+    </IfModule>
+    CustomLog "logs/access.log" common
+</IfModule>
+
+<IfModule alias_module>
+    ScriptAlias /cgi-bin/ "c:\opt\mesosphere\bin\Apache24\cgi-bin\"
+</IfModule>
+
+<IfModule cgid_module>
+    #
+    # ScriptSock: On threaded servers, designate the path to the UNIX
+    # socket used to communicate with the CGI daemon of mod_cgid.
+    #
+    #Scriptsock cgisock
+</IfModule>
+
+<Directory "c:\opt\mesosphere\bin\Apache24\cgi-bin">
+    AllowOverride None
+    Options None
+    Require all granted
+</Directory>
+
+<IfModule headers_module>
+    RequestHeader unset Proxy early
+</IfModule>
+
+<IfModule mime_module>
+    TypesConfig conf/mime.types
+    AddType application/x-compress .Z
+    AddType application/x-gzip .gz .tgz
+</IfModule>
+
+<IfModule proxy_html_module>
+Include conf/extra/proxy-html.conf
+</IfModule>
+
+<IfModule ssl_module>
```

Comment:
No need for ifmodule, we will alaways want to include this module.

Reply:
it will be removed.